src/level/mod.rs

```diff
@@ -300,17 +300,16 @@ impl LevelData {
                 if delta != 0 {
                     // average between two texels
                     let mat = avg(color[3], color[7]);
-                    level.meta[i + 0] =
```

Comment:
these are all intentional
let's have a crate-level lint allowance for this rule

Reply:
still a problem